### PR TITLE
Nikita/Guide button in Binary is missing once we close it. 

### PR DESCRIPTION
**Pull request by nikita-deriv**

### CODE REVIEW

File: src/javascript/app/common/guide.js [hunk 1]
```diff
@@ -43,16 +44,12 @@ const Guide = (() => {
...
         makeButton();
     };
 
-    /*
-     *  do not show the guide button if its close (X) has been clicked before
-     */
     const isDisabled = () => {
         const disabled = Cookies.get(cookie_name);
-        return !!disabled && $.inArray(opt.script, disabled.split(',')) >= 0;
+        return !disabled && $.inArray(opt.script, disabled.split(',')) >= 0;
```
Comment (negarn) [inline src/javascript/app/common/guide.js:52]: we still want to hide it for the current user. just after logging out and logging in we want it to show again. also now for logged out clients it shows a grey box on the trade page.

Comment (nikita-deriv) [reply src/javascript/app/common/guide.js]: I implemented this kind of behaviour and updated code.

File: src/javascript/_common/base/login.js [hunk 2]
```diff
@@ -4,13 +4,16 @@ const getLanguage         = require('../language').get;
 const isMobile            = require('../os_detect').isMobile;
 const isStorageSupported  = require('../storage').isStorageSupported;
 const LocalStore          = require('../storage').LocalStore;
+const removeCookies       = require('../storage').removeCookies;
 const urlForCurrentDomain = require('../url').urlForCurrentDomain;
 const isLoginPages        = require('../utility').isLoginPages;
 const TrafficSource       = require('../../app/common/traffic_source');
+
```
Comment (negarn) [inline src/javascript/_common/base/login.js:11]: empty line?

File: src/javascript/app/base/client.js [hunk 3]
```diff
@@ -116,10 +116,10 @@ const Client = (() => {
     });
 
     const doLogout = (response) => {
-
+
```
Comment (negarn) [inline src/javascript/app/base/client.js:119]: empty tab?

File: src/javascript/app/common/guide.js [hunk 4]
```diff
@@ -38,18 +39,14 @@ const Guide = (() => {
         if ($(opt.guideBtnID).length === 0 || opt.script.length === 0) {
             return;
         }
-
+
```
Comment (negarn) [inline src/javascript/app/common/guide.js:42]: same here

File: src/javascript/app/common/guide.js [hunk 5]
```diff
@@ -38,18 +39,14 @@ const Guide = (() => {
...
             $(opt.guideBtnID).remove();
             return;
         }
-
         makeButton();
     };
-
-    /*
-     *  do not show the guide button if its close (X) has been clicked before
-     */
```
Comment (negarn) [inline src/javascript/app/common/guide.js:52]: I think we don't need to remove this comment now

File: src/javascript/app/base/client.js (hunk 3 above)
Comment (nikita-deriv) [reply src/javascript/app/base/client.js]: Removed. I used eslint to format code, but strangely it left spaces.

File: src/javascript/app/common/guide.js (hunk 4 above)
Comment (nikita-deriv) [reply src/javascript/app/common/guide.js:42]: Removed.

File: src/javascript/app/common/guide.js (hunk 5 above)
Comment (nikita-deriv) [reply src/javascript/app/common/guide.js:52]: Returned.

File: src/javascript/_common/base/login.js (hunk 2 above)
Comment (nikita-deriv) [reply src/javascript/_common/base/login.js]: Removed.

File: src/javascript/app/common/guide.js (hunk 4 above)
Comment (akmal-binary) [reply src/javascript/app/common/guide.js:42]: I can still see the empty tabs and lines here

Comment (nikita-deriv) [reply src/javascript/app/common/guide.js]: I hope i removed it)

File: src/javascript/app/base/client.js [hunk 6]
```diff
@@ -116,10 +116,10 @@ const Client = (() => {
...
```
Comment (negarn) [inline src/javascript/app/base/client.js:119]: could you remove this tab here?

Comment (nikita-deriv) [reply src/javascript/app/base/client.js]: Thanks, removed.

**Review by akmal-binary — APPROVED** [review]

**Review by negarn — APPROVED** [review]

**Pull request merged**